ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -43,3 +46,40 @@ tests:
   commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-frontend.sh
   container:
     from: test-bin
+
+#  builder_run 'GUI-Tests' 1 ./test-gui.sh crud
+- artifact_dir: /tmp/artifacts
+  as: GUI-Tests
+  commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-gui.sh crud
```

Comment:
atm assuming so, as `test_binary_build_commands: ./build.sh` runs both `build-frontend.sh` and `build-backend.sh`, but will see soon.

Reply:
`openshift_ansible_src` runs your `src` container unfortunately, not the built one. @bbguimaraes do we have a plan for that approach?

Reply:
> openshift_ansible_src runs your src container unfortunately

Is that the first container in the multi stage build? I think that is OK here actually

Reply:
It should be the build root with your source cloned into it

Reply:
We don't, that would require a separate template (it is the reason we have the `*-e2e`/`*-src` separation). I'll add that to our backlog.

As a temporary workaround, you could add the `test_binary_build_commands` before the test command. It's inefficient, but it may not add much in comparison to the time it takes to bring the cluster up and down.

Reply:
https://jira.coreos.com/browse/DPTP-280